src/event.rs

```diff
@@ -119,9 +118,11 @@ impl<'nvml> EventSet<'nvml> {
 /// method on the `EventSet` struct if you care about handling them.
 impl<'nvml> Drop for EventSet<'nvml> {
     fn drop(&mut self) {
-        #[allow(unused_must_use)]
+        let sym = nvml_sym(self.nvml.lib.nvmlEventSetFree.as_ref()).unwrap();
```

Comment:
Since we end up calling `unwrap` here anyway, we might as well use the convenience accessors bindgen created for us (`self.nvml.lib.nvmlEventSetFree()`).